### ISSUE TITLE
Inference speed problem on my own environment

### ISSUE THREAD

**Issue by yhfeng1995** [dbolya/yolact#10]
First of all, thanks for sharing the amazing work!
Following the instructions, I have deployed the environment and can execute the code successfully, however, when running eval.py, the inference speed is slower than expected. 
For model ResNet101-FPN, when testing on validation set of coo, the code return about 9 FPS, and when testing on my own images of kinect (640*480),  with ploting and saving disabled, the code return about 14 FPS.
my own evironment is : GTX1080, cuda8.0, cudatoolkits8.0, I am using anaconda, gpu support is checked via
```pytthon
torch.cuda.is_available()
```
I am a newer for pytorch, so I am wondering there is some configuration or dependencies have missed.

Thanks!

**Comment by dbolya** [comment]
We did our timing on a Titan Xp, which should be quite a bit better than a 1080, but 9 fps does sound abnormally low. Here's a couple things I can think of:
 - Do you have CuDNN installed properly? Check if `torch.backends.cudnn.enabled` is True.
 - What's your CPU and OS? Unfortunately with Python's GIL, running the model is more cpu-bound than I would like it to be (we're trying to fix this by implementing a benchmark version), and I find that Pytorch runs slower than expected on my Windows machine vs a Linux one.
 - When you say you test on coco's validation set, is that with `python eval.py` or with `python eval.py --benchmark`? The timing for the former includes computing the mAP and loading data, while the latter only includes the raw model.

**Comment by yhfeng1995** [comment]
Thanks for your quick and detailed reply!
After testing it does caused by the problem of missing argument "--benchmark", after adding this argument, the code return the correct result of about 21 FPS.
Again, thanks for your awesome work :)

**Comment by dbolya** [comment]
Glad to see that worked!
